docs/connect/index.md

```diff
@@ -2,17 +2,32 @@
 
 # Connect to a CrateDB cluster
 
-This documentation section is about connecting your applications and drivers to CrateDB.
+This documentation section is about connecting your applications to CrateDB
+and CrateDB Cloud, using database drivers, and compatibility-adapters and
+-dialects.
 
 
-## Configure
+## Protocol Support
+
+CrateDB supports both the [HTTP protocol] and the [PostgreSQL wire protocol],
+which ensures that many clients that work with PostgreSQL, will also work with
+CrateDB. Through corresponding drivers, CrateDB is compatible with [ODBC],
+[JDBC], [ADBC], and other database API specifications.
```

Comment:
I wasn't aware of ADBC. When searching on this page for JDBC/ODBC, there are links with further details, but ADBC isn't mentioned a second time. Do we have any resources we can mention for interested users?

Reply:
Hi Niklas. You are absolutely right, I expected you to spot that flaw, where the documentation is too flat, yet. ;]

I think [Polars uses ADBC](https://docs.pola.rs/user-guide/io/database/#adbc) already, we added relevant patches already, ....

- https://github.com/crate/cratedb-examples/pull/229
- https://github.com/crate/crate-clients-tools/pull/65

... and there will be a few more in that direction. So, if I am not completely wrong on this topic, there would "just" be a need to cross-reference both topics in a better way, in order to "unflatten" the documentation? Please correct me if I am wrong, @surister.

If you agree 👍, and if you think it is acceptable, I will carry that over into a note to be handled on behalf of a subsequent iteration on the documentation. Otherwise, please tell me to remove the ADBC link again. Thanks!

Reply:
You are right, but tbh I'm thinking that we should perhaps skip the ADBC stuff for now, it's just too green.

Reply:
> we should perhaps skip the ADBC stuff for now

All right, I will remove that item again.

> adding a note that explicitly states that it might stop working at any point.

Aha? All right, we can elaborate about this in another setting.

Thanks.

Reply:
The reference to ADBC has been removed.